ci-operator/step-registry/upi/deprovision/aws/upi-deprovision-aws-commands.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+function queue() {
+    local TARGET="${1}"
+    shift
+    local LIVE
+    LIVE="$(jobs | wc -l)"
+    while [[ "${LIVE}" -ge 45 ]]; do
+    sleep 1
+    LIVE="$(jobs | wc -l)"
+    done
+    echo "${@}"
+    if [[ -n "${FILTER:-}" ]]; then
+    "${@}" | "${FILTER}" >"${TARGET}" &
+    else
+    "${@}" >"${TARGET}" &
+    fi
+}
+
+set +e
+export PATH=$PATH:/tmp/shared
+AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/.awscred
+export AWS_SHARED_CREDENTIALS_FILE
+
+echo "Deprovisioning cluster ..."
+# FIXME: picking up awscli installed by the setup container
+export HOME=/tmp/shared
+export PATH="${HOME}/.local/bin:${PATH}"
+
+AWS_DEFAULT_REGION=$(cat ${SHARED_DIR}/AWS_REGION)  # CLI prefers the former
+export AWS_DEFAULT_REGION
+CLUSTER_NAME=$(cat ${SHARED_DIR}/CLUSTER_NAME)
+
+aws cloudformation describe-stack-resources --stack-name "${CLUSTER_NAME}-control-plane" \
+    --query 'StackResources[?ResourceType==`AWS::EC2::Instance`].PhysicalResourceId' --output text | sed 's,\t,\n,g' > /tmp/node-provider-IDs
+for INDEX in 0 1 2
+do
+    aws cloudformation describe-stack-resources --stack-name "${CLUSTER_NAME}-compute-${INDEX}" \
+    --query 'StackResources[].PhysicalResourceId' --output text | cut -d, -f1 >> /tmp/node-provider-IDs
+done
+
+while IFS= read -r i; do
+    mkdir -p "${SHARED_DIR}/nodes/${i}"
+    queue ${SHARED_DIR}/nodes/$i/console aws ec2 get-console-output --instance-id "${i}" --output text
```

Comment:
This is failing due to a non-ascii horizontal ellipsis in the console output. I don't think we need to block this PR to fix this, but we should at least file a card to fix it later.